src/network/core_utils.rs

```diff
@@ -911,7 +922,16 @@ impl CoreUtils {
                 ))
             }
         }
-
+        let k = format!("/proc/sys/net/ipv6/conf/{}/enhanced_dad", &host_veth);
```

Comment:
Same I dont think we should do a `sysctl` while configuring `veth`

Reply:
I think we should have a different function for this and call `sysctl` in `core.rs`

Reply:
After checking this , we already have `host_veth` at higher level so this better in `core.rs` where we are calling function to configure bridge.